knimin/lib/data_access.py

```diff
@@ -1685,6 +1685,34 @@ def add_external_survey(self, survey, description, url):
                  RETURNING external_survey_id"""
         return self._con.execute_fetchone(sql, [survey, description, url])[0]
 
+    def list_ag_surveys(self, selected=None):
+        """Returns the list of american gut survey names
+
+        Parameters
+        ----------
+        selected : list of int
+            To keep track of which surveys have been selected by the user via
```

Comment:
`the resulting list of tupels third element holds this information as a boolean value` -> I don't understand what this means, can you rephrase?